### PR TITLE
adjusted sonar scanner playbook and created from main branch.

### PR DESCRIPTION
**Pull request by samirparhi-dev**
please merge it if it is okay

### CODE REVIEW

File: playbooks/besman-sast-sonarqube-0.0.1-playbook.sh
Comment (asa1997) [inline playbooks/besman-sast-sonarqube-0.0.1-playbook.sh:1]: Please rename the file name to besman-sonarqube-0.0.1-playbook.sh

**Review by asa1997 — CHANGES_REQUESTED** [review]
@samirparhi-dev Please address the change

File: playbooks/besman-sast-sonarqube-0.0.1-playbook.sh
Comment (samirparhi-dev) [reply playbooks/besman-sast-sonarqube-0.0.1-playbook.sh:1]: done

**Review by asa1997 — CHANGES_REQUESTED** [review]

File: playbook-metadata.json
Comment (samirparhi-dev) [reply playbook-metadata.json:44]: Plz you do the changes, I will be not be able do the minor changes. Else assignment some one else to do these changes. Until unless there is a common understanding and clarity on documentation I can not do it again and again.

File: playbooks/besman-sonarqube-0.0.1-playbook.sh
Comment (samirparhi-dev) [reply playbooks/besman-sonarqube-0.0.1-playbook.sh:13]: Plz you do the changes, I will be not be able do the minor changes. Else assignment some one else to do these changes. Until unless there is a common understanding and clarity on documentation I can not do it again and again.

File: playbooks/besman-sonarqube-0.0.1-playbook.sh
Comment (samirparhi-dev) [reply playbooks/besman-sonarqube-0.0.1-playbook.sh:15]: Plz you do the changes, I will be not be able do the minor changes. Else assignment some one else to do these changes. Until unless there is a common understanding and clarity on documentation I can not do it again and again.

File: playbooks/besman-sonarqube-0.0.1-playbook.sh
Comment (samirparhi-dev) [reply playbooks/besman-sonarqube-0.0.1-playbook.sh:51]: Plz you do the changes, I will be not be able do the minor changes. Else assignment some one else to do these changes. Until unless there is a common understanding and clarity on documentation I can not do it again and again.

**Comment by asa1997** [comment]
@samirparhi-dev These errors were found out when I executed your playbook. You should have an understanding on how these scripts work, only then you could avoid these minor changes. If you had any queries, you could have asked them before you raised the pr or before you started working on these and its not ethical to drop something, which was  assigned to you, soo close to the deadline.

So, I kindly request you to finish the task assigned to you on time.

@panickervinod @sumodgeorge @harimohanr

**Comment by samirparhi-dev** [comment]
@sumodgeorge <https://github.com/sumodgeorge> : Kindly have a look in to
below points accordance to above reply:

- As per the above reply from the maintainer, The open source Community
trying to teach me ethics where they don't follow it. Few Examples:
                   - There is no process defined or documented
                   - The design spec is not documented any where. Whatever
comes to the mind to the maintainer , he thinks that that is a feature
without taking approval from community member.
                 - Release happens whenever they want
                 - Reluctant to adopt changes
- This tool does not have compatibility.
        Example:
          - This tool is claimed to be only designed to works on Ubuntu. I
have multiple times iterated we have verity of OS  still the maintainer
expect me to perform test before commits.
            - In the same line I have highlighted  our team uses a wide
range of OS, so the maintainer needs an approval while doing release which
is not taken care of yet.
         -
          - Maintainer is reluctant to make even small changes which need a
little effort and expect to be done by the original commuter which shows
they are not very serious about features.

Currently I have more other priorities to work on rather than contributing
to this kind of an Community which is very much monotonous

 @panickervinod <https://github.com/panickervinod>   @harimohanr
<https://github.com/harimohanr>   FYI
-----------------------------------------------------------------------
*Samir Ranjan Parhi*
*Email:* ***@***.***
*Website:* https://ls-lrt.com
*GitHub:* https://github.com/samirparhi-dev
*LinkedIn:* https://www.linkedin.com/in/samir-parhi

**Comment by harimohanr** [comment]
@samirparhi-dev We have reused some of the files you have implemented and merged it into develop branch in a seperate PR. Closing this PR as it is stale currently. Thanks Samir for this contribution.

**Pull request closed without merging**
